app/controllers/kudos_controller.rb

```diff
@@ -75,17 +63,15 @@ def create
     # by database unique indices, use the usual duplicate error message.
     #
     # https://api.rubyonrails.org/v5.1/classes/ActiveRecord/Validations/ClassMethods.html#method-i-validates_uniqueness_of-label-Concurrency+and+integrity
+    error_message = t("activerecord.errors.models.kudo.taken")
```

Comment:
This is the error key of the `uniqueness` validations, which we will need to customize in any case. I use it here to avoid adding another key for the controller message. But I can add a `duplicate` key if you prefer.

Reply:
Nah, let's keep it to just the one key since we don't have a choice.